tests/unit/conftest.py

```diff
@@ -46,6 +46,7 @@ def current_global_defaults():
         "shape": "Default",
         "users": 1.0,
         "runtime": 120.0,
+        "stop_timeout": 300.0,
```

Comment:
5 minutes for a graceful shutdown of users seems very long? at least for a default value.

i can't seem to find where this value is being used for anything? where is the code for that?

Reply:
I did remove the test arg from this PR.